jest/defs.bzl

```diff
@@ -105,21 +120,33 @@ def jest_test(
 
             See https://jestjs.io/docs/configuration#testsequencer-string for more information on Jest testSequencer config option.
 
-        snapshot_ext: The expected extensions for snapshot files. Defaults to `.snap`, the Jest default.
+        snapshots_ext: The expected extensions for snapshot files. Defaults to `.snap`, the Jest default.
+
+        silent_on_success: Whether the snapshot update generator target should write to stdout & stderr on success.
```

Comment:
the text seems logical NOT of silent, it describes noisy_on_success

Reply:
Update the attribute name and docstring. See comment above.